modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -125,6 +129,7 @@ NOTE: The `mcrypt` library is deprecated and removed from the PHP 7.2 core. Howe
 === php-libsodium
 
 If you see the message `PHP Fatal error: sodium_init() in Unknown on line 0` when invoking the command `php -v` or when running PHP programs, `php-libsodium` might be installed.
```

Comment:
This sentence should end with "might not be installed".

Reply:
Just cross read it.
might not be installed is NOT correct. When you eg upgrade from an older Ubuntu version and have a php release lower than 7.2, you possibly will have php-libsodium installed. Now, when upgrading to 7.2, you have an "overinstalled" situation - you can safely remove php-libsodium as described.